zppy/global_time_series.py

```diff
@@ -52,6 +53,48 @@ def global_time_series(config, scriptDir, existing_bundles, job_ids_file):
             if skip:
                 continue
 
+            # Handle legacy parameter
+            if (not c["plots_original"]) and c["plot_names"]:
+                c["plots_original"] = c["plot_names"]
+
+            # Determine which components are needed
+            c["use_atm"] = False
+            c["use_lnd"] = False
+            c["use_ocn"] = False
+            c["use_sea_ice"] = False
+            if c["plots_original"]:
+                c["use_atm"] = True
+                if not c["atmosphere_only"]:
+                    c["use_ocn"] = True
+            else:
+                c[
+                    "plots_original"
+                ] = "None"  # For better string processing in global_time_series.bash
+            if c["plots_atm"]:
+                c["use_atm"] = True
+            else:
+                c[
+                    "plots_atm"
+                ] = "None"  # For better string processing in global_time_series.bash
```

Comment:
Just a comment here -- `global_time_series` is particularly cumbersome for passing parameters since it essentially involves a python -> bash -> python process. `global_time_series.py` launches `global_time_series.bash` which launches `coupled_global.py`. This means parameters have to be shuffled between languages that handle variables differently.